packages/ui/test/working-groups/modals/AddWorkerStakeModal.test.tsx

```diff
@@ -0,0 +1,194 @@
+import { cryptoWaitReady } from '@polkadot/util-crypto'
+import { configure, fireEvent, render, screen } from '@testing-library/react'
+import React from 'react'
+
+import { Account } from '@/accounts/types'
+import { ApiContext } from '@/common/providers/api/context'
+import { ModalContext } from '@/common/providers/modal/context'
+import { AddWorkerStakeModal } from '@/working-groups/modals/AddStakeModal'
+
+import { getButton } from '../../_helpers/getButton'
+import { alice, bob } from '../../_mocks/keyring'
+import { MockKeyringProvider, MockQueryNodeProviders } from '../../_mocks/providers'
+import { setupMockServer } from '../../_mocks/server'
+import {
+  stubApi,
+  stubDefaultBalances,
+  stubTransaction,
+  stubTransactionFailure,
+  stubTransactionPending,
+  stubTransactionSuccess,
+} from '../../_mocks/transactions'
+
+configure({ testIdAttribute: 'id' })
+
+const useMyAccounts: { hasAccounts: boolean; allAccounts: Account[] } = {
+  hasAccounts: true,
+  allAccounts: [],
+}
+
+jest.mock('@/accounts/hooks/useMyAccounts', () => {
+  return {
+    useMyAccounts: () => useMyAccounts,
+  }
+})
+
+describe('UI: AddStakeModal', () => {
+  setupMockServer()
+  const api = stubApi()
+  let transfer: any
+
+  const mockModalContext = {
+    hideModal: jest.fn(),
+    modalData: {
+      worker: {
+        minStake: 300,
+        stake: 100,
+        roleAccount: alice.address,
+        id: 'workerId',
+        group: {
+          id: 'storageWorkingGroup',
+        },
+        runtimeId: 88,
+      },
+    },
+    modal: null,
+    showModal: jest.fn(),
+  }
+
+  beforeAll(async () => {
+    await cryptoWaitReady()
+    useMyAccounts.allAccounts.push(alice, bob)
+  })
+
+  beforeEach(async () => {
+    stubDefaultBalances(api)
+    transfer = stubTransaction(api, 'api.tx.storageWorkingGroup.increaseStake')
+  })
+
+  describe('Form step', () => {
+    it('Renders modal', () => {
+      const { queryByText } = renderModal()
+
+      expect(queryByText('Add stake')).toBeDefined()
```

Comment:
```suggestion
     renderModal()

      expect(screen.queryByText('Add stake')).toBeDefined()
```
We have some mixup in the tests, but I think that we should stick to one way of querying (by using `screen`). This makes changing query methods faster (no need to update the destructuring object returned from `renderModal()`.